server/src/routes/internship.ts

```diff
@@ -105,11 +110,35 @@ internshipRouter.patch(
   "/:id",
   authMiddleware(),
   param("id").custom(isObjectId),
-  query(standardPostParams),
+  body(standardPostParams),
   validate,
   asyncHandler(updateInternship)
 );
 
+internshipRouter.delete(
+  "/:id",
+  authMiddleware(),
+  param("id").custom(isObjectId),
+  validate,
+  asyncHandler(deleteInternship)
+);
+
+internshipRouter.patch(
+  "/:id/approve",
```

Comment:
mich stört noch ein bisschen dass in der url etwas anderes als nur entities stehen (also auch ein verb) aber ich wüsste gerade auch nicht wie man's anders machen könnte. höchstens vlt `?status=approve". Hm.

Reply:
Ok :)